example_site/src/components/AccountInfoForm.js

```diff
@@ -1,16 +1,30 @@
 import { useForm } from "react-hook-form";
+import React, { useState } from "react";
 
 import { useAppContext } from "./AppProvider";
 
 export default function AccountInfoForm({
   account,
-  onSubmit,
+  updateAccount,
   formTitle,
   subtitle,
   submitButtonText,
+  canUpdate,
```

Comment:
I think this could be a little more broad as well - `canUpate` only ends up helping to control the button's `disabled` flag, but really what it's saying is, "can the form be submitted if there were no changes to the input"? We could use this prop to set the initial `buttonStatus` value. 

Note for myself to bring up during our 1:1...
- Reusable components should not be opinionated